aodndata/moorings/products_handler.py

```diff
@@ -180,10 +182,13 @@ def _get_input_files(self):
         # TODO: Replace temp_dir above with cache_dir?
 
     def _get_old_product_files(self):
-        """Get a list of the currently published aggregated_timeseries files for the site being processed."""
+        """Get a list of the currently published product files for the site being processed."""
 
         filter_list = [PropertyIsEqualTo(propertyname='site_code', literal=self.product_site_code),
```

Comment:
since this is very chatty, I would create two simple variables:
```
match_site_code = PropertyIsEqualto(...)
match_product_category = Or([...])
filter_list = [match_site_code, match_products_category]
```

Reply:
DOne